### PR TITLE
Fix Link to Policy Docs

### PR DESCRIPTION
**Pull request by laurapacilio**
### What
After we redid the policy updates, some old pages still have direct links to the sentinel content. When I initially redirected, I redirected to the new policy enforcement overview page, which talks about the entire policy enforcement workflow and mentions both sentinel and OPA policies.

I actually think this is a reasonable redirect, because 1) The old /sentinel page was an overview of policy enforcement (similar to the new page) it's just that policy enforcement used to be sentinel-only and 2) I want to start helping folks understand there are both types of policies available.

Regardless, This PR:
- Updates the link text on this page to include mention of OPA .
- Removes the [sentinel] link at the top and replacing it with a direct link in the text because I think it's much easier to evaluate things like this in the future when you think about each link in context, rather than just redirecting it at the top of the page like that.
- Reworks the Policies section of this page to include OPA and be less Sentinel-centric. 

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/run/cli.mdx [hunk 1]
```diff
@@ -27,7 +25,7 @@ Terraform Cloud has three workflows for managing Terraform runs.
 
 The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It offers the best of both worlds to developers who are already comfortable with using the Terraform CLI, and it can work with existing CI/CD pipelines.
 
-You can start runs with the standard `terraform plan` and `terraform apply` commands and then watch the progress of the run from your terminal. These runs execute remotely in Terraform Cloud; they use variables from the appropriate workspace, enforce any applicable [Sentinel policies][sentinel], and can access Terraform Cloud's [private registry][private] and remote state inputs.
+You can start runs with the standard `terraform plan` and `terraform apply` commands and then watch the progress of the run from your terminal. These runs execute remotely in Terraform Cloud; they use variables from the appropriate workspace, enforce any applicable [Sentinel or OPA policies][/cloud-docs/policy-enforcement], and can access Terraform Cloud's [private registry][private] and remote state inputs.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/run/cli.mdx:28]:
```suggestion
You can start runs with the standard `terraform plan` and `terraform apply` commands and then watch the progress of the run from your terminal. These runs execute remotely in Terraform Cloud, use variables from the appropriate workspace, enforce any applicable [Sentinel or OPA policies][/cloud-docs/policy-enforcement], and can access Terraform Cloud's [private registry][private] and remote state inputs.
```

Suggest removing semicolon and forming 1 sentence. If 1 sentence is too long / a run-on, suggest splitting into 2 sentences.

File: website/docs/cloud-docs/run/cli.mdx [hunk 2]
```diff
@@ -27,7 +25,7 @@ Terraform Cloud has three workflows for managing Terraform runs.
 
 The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It offers the best of both worlds to developers who are already comfortable with using the Terraform CLI, and it can work with existing CI/CD pipelines.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/run/cli.mdx:26]:
```suggestion
The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It stronger functionality developers who are already comfortable with using the Terraform CLI and works with existing CI/CD pipelines.
```

Maybe just a personal thing, but saying "It offers the best of both worlds" feels like marketing speak / overly colloquial language to me. Feel free to ignore if you disagree.

**Review by mgarrell777 — APPROVED** [review]
@laurapacilio Thanks very much for this PR. I have a couple minor nits, but looks good to me overall.

File: website/docs/cloud-docs/run/cli.mdx (hunk 2 above)
Comment (laurapacilio) [reply website/docs/cloud-docs/run/cli.mdx:26]: this is outside the scope of this PR, so I think I'm going to ignore for now :)

**Pull request merged**